LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua

```diff
@@ -349,9 +349,9 @@ local localization = {
         TRANSLATION = "Translation",
         DONATION = "Donate",
         PANEL_INFO_FONT = "$(CHAT_FONT)|14|soft-shadow-thin",
-        RELOAD_UI_WARNING = "Changes to this setting require an UI reload in order to take effect.",
-        RELOAD_DIALOG_TITLE = "UI Reload required",
-        RELOAD_DIALOG_TEXT = "Some changes require an UI reload in order to take effect. Do you want to reload now or discard the changes?",
+        RELOAD_UI_WARNING = GetString(SI_OPTIONS_APPLY_WARNING),
+        RELOAD_DIALOG_TITLE = GetString(SI_ADDON_MANAGER_RELOAD),
+        RELOAD_DIALOG_TEXT = "Some changes require a UI reload in order to take effect. Do you want to reload now or discard the changes?",
```

Comment:
Are you sure about this? I am not a native speaker, but I am pretty certain that it is "an UI reload".

Reply:
I am sure. The rule is "how it sounds" not "which letter." For any English speaker, reading that aloud will sound very wrong.

https://www.englishforums.com/English/AUtilityVsAnUtility/hrmlw/post.htm

Reply:
Guess you are right. The more you know. :D